exercises/bank.js

```diff
@@ -1,51 +1,88 @@
 
 
 class Bank{
-	constructor(){
+	constructor(bankName){
 		//takes in the name of the bank.
 		//makes an object to store all accounts into
+		this.name = bankName;
+		this.accountList = {};
 	}
-	makeAccount( ){
+	makeAccount(accountNumber){
 		//makes an account
+
 		//takes in the account number of an account
 		//checks if the account number already exists in the account storage object in the constructor
 			//if already exists, returns false
 		//otherwise makes an new Account object, storing it into the storage object in the constructor
 		//returns the Account object that was constructed
+		if (this.checkForAccount(accountNumber)) {
+			return false;
+		} else {
+			var newAccount = new Account();
+			this.accountList[accountNumber] = newAccount;
+			return newAccount;
+		}
 	}
-	checkForAccount(  ){
+	checkForAccount(accountNumber){
```

Comment:
Functions that are included in a "class constructor" are called methods, and will be attached to any object that is created by the "class constructor". This is incredibly useful for easily adding functionality to objects with the ease of an assembly line. Imagine creating a "person class constructor" that needed to have methods such as "eating", "drinking", and "studying". By doing this, every student that was created by the "class constructor" would be able to eat, drink, and study (I would say "sleep" would be a good method, but most students don't get to do that). 
The utility of this is easy to understand, as it allows code to be easily replicated in each new object simply by calling the "class constructor" Great Job!